### PR TITLE
Kyma version as label and disclaimer-warning as annotation

### PR DESCRIPTION
**Pull request by rJankowski93**
https://github.com/kyma-incubator/reconciler/issues/170

### CODE REVIEW

File: pkg/reconciler/service/annotationsinterceptor.go [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package service
+
+import (
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+const (
+	ManagedByAnnotation       = "reconciler.kyma-project.io/managed-by-reconciler-disclaimer"
+	AnnotationReconcilerValue = "DO NOT EDIT - This resource is managed by Kyma.\nAny modifications are discarded and the resource is reverted to the original state."
```
Comment (tobiscr) [inline pkg/reconciler/service/annotationsinterceptor.go:9]:
```suggestion
	annotationReconcilerValue = "DO NOT EDIT - This resource is managed by Kyma.\nAny modifications are discarded and the resource is reverted to the original state."
```

Comment (tobiscr) [reply pkg/reconciler/service/annotationsinterceptor.go:9]: The Key makes sense to have it public, but would make the value private.

File: pkg/reconciler/service/annotationsinterceptor.go [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+type AnnotationsInterceptor struct {
+}
+
+func (l *AnnotationsInterceptor) Intercept(resource *unstructured.Unstructured) error {
+	annotations := resource.GetAnnotations()
+	if annotations == nil {
+		annotations = make(map[string]string)
+	}
+	annotations[ManagedByAnnotation] = AnnotationReconcilerValue
```
Comment (tobiscr) [inline pkg/reconciler/service/annotationsinterceptor.go:20]:
```suggestion
	annotations[ManagedByAnnotation] = annotationReconcilerValue
```

File: pkg/reconciler/service/annotationsinterceptor_test.go [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+							},
+						},
+					},
+				},
+			},
+			wantErr: false,
+			annotations: map[string]string{
+				"some-annotation":   "some-value",
+				"some-annotation2":  "some-value2",
+				ManagedByAnnotation: AnnotationReconcilerValue,
```
Comment (tobiscr) [inline pkg/reconciler/service/annotationsinterceptor_test.go:49]:
```suggestion
				ManagedByAnnotation: annotationReconcilerValue,
```

**Review by tobiscr — CHANGES_REQUESTED** [review]

File: pkg/reconciler/service/annotationsinterceptor_test.go [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+		annotations map[string]string
+	}{
+		{
+			name: "Resource without any annotations",
+			args: args{
+				resource: &unstructured.Unstructured{},
+			},
+			wantErr: false,
+			annotations: map[string]string{
+				ManagedByAnnotation: AnnotationReconcilerValue,
```
Comment (tobiscr) [inline pkg/reconciler/service/annotationsinterceptor_test.go:26]:
```suggestion
				ManagedByAnnotation: annotationReconcilerValue,
```

**Review by tobiscr — APPROVED** [review]

**Review by suleymanakbas91 — APPROVED** [review]

**Pull request merged**